jupyter-book/spatial/neighborhood.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    sq.pl.nhood_enrichment(adata, cluster_key="cluster", method="average", figsize=(5, 5))
</pre><p>I wish someone woulf fix the hierarchical bar on top of the heatmap, it's quite annoying, but alas...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>